Tests/ApolloTests/Cache/StoreSubscriptionTests.swift

```diff
@@ -1,24 +1,62 @@
 import Nimble
 import XCTest
 @testable import Apollo
+//@testable import ApolloSQLite
 
-class StoreSubscriptionTests: XCTestCase {
+open class StoreSubscriptionTests: XCTestCase {
   static let defaultWaitTimeout: TimeInterval = 1
 
+  var cache: NormalizedCache!
   var store: ApolloStore!
 
-  override func setUpWithError() throws {
+  open override func setUpWithError() throws {
     try super.setUpWithError()
-    store = ApolloStore()
+    cache = InMemoryNormalizedCache()
+    //cache = try! SQLiteNormalizedCache(fileURL: URL(fileURLWithPath: "/tmp/test.sqlite"))
```

Comment:
```suggestion
```

Cleaning this up too. The underlying cache provider shouldn't make a difference in these tests but arguably a memory-based cache would be slightly faster.